superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -0,0 +1,196 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import styled from '@superset-ui/style';
+import Icon from 'src/components/Icon';
+import { Card } from 'src/common/components';
+
+const MenuIcon = styled(Icon)`
+  width: 14px;
+  height: 14px;
+  position: relative;
+  top: 1px;
+`;
+
+const ActionsWrapper = styled.div`
+  width: 64px;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const StyledCard = styled(Card)`
+  width: 459px;
+
+  .ant-card-body {
+    padding: 16px 8px;
+  }
+  .ant-card-meta-detail > div:not(:last-child) {
+    margin-bottom: 0;
+  }
+`;
+
+const Cover = styled.div`
+  height: 264px;
+  overflow: hidden;
+
+  .cover-footer {
+    transform: translateY(36px);
+    transition: 0.2s ease-out;
+  }
+
+  &:hover {
+    .cover-footer {
+      transform: translateY(0);
+    }
+  }
+`;
+
+const GradientContainer = styled.div`
+  position: relative;
+  display: inline-block;
+
+  &:after {
+    content: '';
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    display: inline-block;
+    background: linear-gradient(
+      180deg,
+      rgba(0, 0, 0, 0) 47.83%,
+      rgba(0, 0, 0, 0.219135) 79.64%,
+      rgba(0, 0, 0, 0.5) 100%
+    );
+  }
+`;
+const CardCoverImg = styled.img`
+  display: block;
+  object-fit: cover;
+  width: 459px;
```

Comment:
Could just be 100% width, and auto height if the card has its own width set, I think?

Reply:
Doesn't seem to like that

<img width="485" alt="Screen Shot 2020-08-12 at 5 32 55 PM" src="https://user-images.githubusercontent.com/10255196/90081320-e6f77f80-dcc1-11ea-9ed1-3982f3921778.png">